### ISSUE TITLE
pre-trained weights and seed

### ISSUE THREAD

**Issue by Serendipity-LC** [CVIR/TCL#13]
Thanks for this great repo!

Whether pre-training weights, such as ImageNet, are used in the self-supervised pre-training stage? And whether ImageNet pre-training weights or other pre-training weights are used if [TCL→Finetuning] is executed without self-supervised pre-training?

I noticed that the models in the paper did three randomized trials when they were trained using different percentages of labeled samples, how were the seed selected in these three trials?
